### ISSUE TITLE
System Error when running Ganache 1.1.0 on darwin

### ISSUE THREAD

**Issue by JCCDex** [trufflesuite/ganache#770]
PLATFORM: darwin
GANACHE VERSION: 1.1.0

EXCEPTION:
```
Error: listen EADDRINUSE 127.0.0.1:7545
    at Object.exports._errnoException (util.js:1024:11)
    at exports._exceptionWithHostPort (util.js:1047:20)
    at Server.setupListenHandle [as _listen2] (net.js:1319:14)
    at listenInCluster (net.js:1367:12)
    at doListen (net.js:1493:7)
    at _combinedTickCallback (internal/process/next_tick.js:141:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)
```

APPLICATION LOG:
```
T+0ms: Starting server with initial configuration: {"hostname":"127.0.0.1","network_id":5777,"port":7545,"total_accounts":10,"unlocked_accounts":[],"vmErrorsOnRPCResponse":true}
```

**Comment by jklepatch** [comment]
You might have a rogue instance of ganache still running.

Run this to find the pid of the offending process:

```
lsof -nP -i4TCP:7545
```

Then kill the offending process with:

```
kill -9 {REPLACE_BY_PID_HERE}
```

Then you should be able to run Ganache again.

**Comment by seesemichaelj** [comment]
Thanks for reporting this! This issue is occurring because of a bug leaving the port open sometimes. The issue #665 was resolved in 1.2.x

### Please Update
We updated the interface and made it easier for you to change the settings when they don't work (like this issue).

Make sure all processes of Ganache are stopped (i.e. task manager or system monitor, etc.), and then head over to https://github.com/trufflesuite/ganache/releases/latest to get the most recent version of Ganache!
